data/mods/gen3/abilities.ts

```diff
@@ -84,7 +88,6 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
 				return this.effectState.target;
 			}
 		},
-		isBreakable: true,
```

Comment:
i think you still need to do a flag property here since this entry doesnt inherit

Reply:
Mold Breaker effects didn't exist in Gen 3, but it's not like I'm overriding everything else to not be breakable in Gen 3 so good eye, I'll restore that.